### PR TITLE
Feature/dismiss transitions

### PR DESCRIPTION
**Pull request by RamonGilabert**
With physics:

app

Without physics:

app

### CODE REVIEW

File: Demos/LightboxDemo/LightboxDemo/MainViewController.swift [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import UIKit
+import Lightbox
+
+class MainViewController: UIViewController {
+
+  lazy var galleryButton: UIButton = {
```
Comment (vadymmarkov) [inline Demos/LightboxDemo/LightboxDemo/MainViewController.swift]: `[unowned self] in` needs to be added here.

File: Source/LightboxTransition.swift [hunk 2]
```diff
@@ -0,0 +1,76 @@
+import UIKit
+
+class LightboxTransition: NSObject {
```
Comment (vadymmarkov) [inline Source/LightboxTransition.swift]: Do we need extend `NSObject`?

File: Source/LightboxTransition.swift [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+  struct Timing {
+    static let Transition: NSTimeInterval = 0.5
```
Comment (vadymmarkov) [inline Source/LightboxTransition.swift]: I think this constant should be non-capital, just `transition`.

File: Source/LightboxTransition.swift [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  }
+
+  var presentingViewController = false
+
+  func transition(controller: LightboxController, show: Bool) {
+    controller.view.backgroundColor = show ? UIColor.blackColor() : UIColor.clearColor()
```
Comment (vadymmarkov) [inline Source/LightboxTransition.swift]: `UIColor` could be dropped.

File: Source/LightboxTransition.swift (hunk 2 above)
Comment (RamonGilabert) [inline Source/LightboxTransition.swift]: Yes, for some reason `UIViewControllerAnimatedTransitioning` needs NSObject, if I delete it it says that it doesn't conform to the protocol...

**Comment by vadymmarkov** [comment]
@RamonGilabert Looks good for me! Should we merge or you said you want to refactor something?

**Comment by RamonGilabert** [comment]
We can merge for now, if I find a better way tomorrow to rewrite the physics and the no physics together, I'll make a PR tomorrow! :smiley_cat:

**Comment by vadymmarkov** [comment]
Sounds like a plan :wink:

**Comment by vadymmarkov** [comment]
giphy1

**Comment by RamonGilabert** [comment]
Jajajaja nice! :)

**Comment by 3lvis** [comment]
Update on

**Pull request merged**